Requests/Builders/CatalogBuilders/ChangeCatalogItemsRequestBuilder.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
```

Comment:
I think it's time to start introducing nullable reference types in the library, let's enable them for the new code